lib/ruby_audit/cli.rb

```diff
@@ -52,6 +57,72 @@ def version
 
     private
 
+    def say(message = '', color = nil)
+      color = nil unless $stdout.tty?
+      super(message.to_s, color)
+    end
+
+    # rubocop:disable Metrics/AbcSize
+    # rubocop:disable Metrics/CyclomaticComplexity
+    # rubocop:disable Metrics/MethodLength
+    # rubocop:disable Metrics/PerceivedComplexity
+    def print_advisory(gem, advisory)
```

Comment:
`say` and `print_advisory` are copied over from bundler-audit v0.7.0.1's `CLI` class: https://github.com/rubysec/bundler-audit/blob/v0.7.0.1/lib/bundler/audit/cli.rb#L97-L157

bundler-audit v0.8.0 gives you a choice between text output and json output, so they moved this logic into some other classes.  Instead of introducing another coupling between the gems, though, I figured I'd just copy the logic back to be consistent with the output we had before.  I'm also in favor of allowing the output formats for ruby_audit and bundler-audit to diverge, so both gems can output results in the best way for the things they test.